### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by YarasKo**

### CODE REVIEW

**Comment by yadamskaya** [comment]
So)
1. Please remove everything that you not use from Servlet files. (variables, method overrides, imports)
2. @WebServlet(name = "Servlet") you are using javax.servlet annotations.  Annotations gives you ability to not register servlet in the web descriptor (web.xml).
So please left something one. Or define everything through annotation or define everything in descriptor and don't use annotations. It will be better to remove annotations.
3. Folder structure isn't good. Java classes should have their package.
Some remarks:
What about the more obvious Java class names? (no Servlet1, Servlet2, Servlet3...)
And please merge previous  PRs to master.

File: task5/F2/src/Servlets/redirectStatus.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package Servlets;
```
Comment (lexastef) [inline task5/F2/src/Servlets/redirectStatus.java:1]: Cosmetic: It's better to not use upper case letters in the package name w/o needs.

File: task5/F2/src/Servlets/redirectStatus.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+
+public class redirectStatus extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```
Comment (lexastef) [inline task5/F2/src/Servlets/redirectStatus.java:11]: Please try leave clear classes.
That servlet is not using doPost method so no reason to leave it.

**Comment by lexastef** [comment]
Mark for the 5th lab updated.

**Pull request merged**